<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiEditClass</class>
 <widget class="QMainWindow" name="QtGuiEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="ShowProcessList">
          <property name="styleSheet">
           <string notr="true">QTableView{


	border-color: rgb(255, 255, 255);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="FlushButton">
          <property name="text">
           <string>刷新进程</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="ShowAdd">
            <property name="styleSheet">
             <string notr="true">QTableView{

	border-color: rgb(255, 255, 255);


}</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>正在进行</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="styleSheet">
               <string notr="true">QProgressBar
{
text-align:center;
	background-color: rgb(0, 0, 127);
}</string>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_id">
            <property name="text">
             <string>输入进程的ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="InputProcessId"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_value">
            <property name="text">
             <string>输入查找的值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="InputFindValue1"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lable_EditValue">
            <property name="text">
             <string>需要修改的值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="InputEditValue"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="CloseButton">
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton">
            <property name="styleSheet">
             <string notr="true">QPushButton{


}</string>
            </property>
            <property name="text">
             <string>查找</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="EditButton_2">
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiEdit.qrc"/>
 </resources>
 <connections/>
</ui>
